### PR TITLE
Release through travis

### PR DESCRIPTION
**Pull request by elcuervo**
- Handles the release responsibility (and credentials) to Travis
- Version bump to match rubygems

### CODE REVIEW

File: imagga_auto_tag.gemspec [hunk 1]
```diff
@@ -6,8 +6,8 @@ require 'imagga_auto_tag/version'
 Gem::Specification.new do |spec|
   spec.name          = 'imagga_auto_tag'
   spec.version       = ImaggaAutoTag::VERSION
-  spec.authors       = ['Luke Chesser']
-  spec.email         = ['luke@unsplash.com']
+  spec.authors       = ['Luke Chesser', 'Unsplash Team']
```
Comment (lukechesser) [inline imagga_auto_tag.gemspec:9]: Feel free to remove me. When I made this, it was just meeeeeee, all alone

**Review by lukechesser — APPROVED** [review]
🎉  much better.

My travis skills are limited — does this only run the release on `master`?

File: imagga_auto_tag.gemspec (hunk 1 above)
Comment (elcuervo) [reply imagga_auto_tag.gemspec:9]: Attribution where attribution is due.

**Comment by elcuervo** [comment]
@lukechesser Yup, only master

**Pull request merged**